docs/reference-implementation/Troubleshooting.md

```diff
@@ -81,6 +81,45 @@ Deployment of service [HealthService | BackgroundProcessor | CatalogService] fai
 
 **Solution:** Re-run the failing step - there's a high probability that it will work. If not on second run, investigate pod health (look at AKS logs, potential error messages from deployment etc.).
 
+---
+
+**Error:** Deploy CatalogService workload / Install workload CatalogService on AKS clusters failed
+
+```console
+certificate for catalogservice-ingress-secret pending.. 
+Deployment not ready. Retrying... 1/5
+..
+Deployment not ready. Retrying... 5/5
+##[error]Script failed with exit code: 1
+```
+
+**Description:** The "Install workload CatalogService on AKS clusters" step actively monitors the provisioning of a LetsEncrypt certificate for the ingress during the installation of the CatalogService helm chart. If the certificate is not ready, or cannot successfully provisioned this task will fail.
+
+**Solution:** This is most of the time caused by hitting LetsEncrypt thresholds (see [Rate Limits](https://letsencrypt.org/docs/rate-limits/) for more details) for certificate provisioning. You can try to re-run the step, or wait for the certificate to be ready. The following commands can help to investigate the issue further:
+
+```console
+# Check certificate status
+kubectl get certificates -n workload
+
+# Check certificate request status
+kubectl get certificaterequests -n workload
+
+# Check cert-manager logs for more details
+kubectl logs deploy/cert-manager -n cert-manager 
+```
+
+Relevant log events are for example `cert-manager/certificates-trigger "msg"="Not re-issuing certificate as an attempt has been made in the last hour" "key"="workload/catalogservice-ingress-secret" "retry_delay"=3599055978096`.
+
+To manually force cert-manager to re-request the certificate you can delete the existing one:
+
+```console
+# Check certificate status
+kubectl get certificates -n workload
+
```

Comment:
I guess we dont have a sample console output right now, that we could show?! maybe lets keep that in mind and add once we run into it again

Reply:
Right - I've already manually fixed and destroyed the affected environment.